css/style.css

```diff
@@ -1,6 +1,163 @@
 /* General page styles */
 
-/* We should use a mobile first approach having out breack point at 768px */
+/* We should use a mobile first approach having out brake point at 768px */
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 0;
+	padding: 0;
+}
+
+.container {
+	position: relative;
+	margin: auto;
+	max-width: 960px;
+	height: 100%;
+}
+
+.main-nav {
+	position: fixed;
+	top: 0;
+	z-index: 2;
+	width: 100%;
+	height: 75px;
+	overflow-x: hidden;
+	transition: 0.5s;
+}
+
+.colored__main-nav {
+	background-color: rgba(0, 60, 143, 0.8);
+}
+
+.transparent__main-nav {
+	background-color: rgba(0, 60, 143, 0);
+}
+
+.main-nav__logo {
+	position: absolute;
+	top: 11px;
+	left: 20px;
+}
+
+.main-nav__logo img {
```

Comment:
please no use tag for assign styles, only classes

Reply:
Fixed. Won't do it again.